CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,25 @@
+## General communication guidelines
+
+Ulauncher is a free, open source software project developed by a small team of volunteers and contributors.
+
+We are here because we want Ulauncher to improve, and for that we have to:
+* Act respectful towards each other.
+* Give each other the benefit of the doubt.
+* Give and accept constructive feedback.
+* Avoid arguments.
+* Learn from past mistakes.
+* Focus on what is best, not just for us as individuals, but for the Ulauncher community.
+* Refrain from trying to "win" arguments or advocate the importance of ones own specific needs.
+* Leave aside anything that is unhelpful for the Ulauncher progress, such as politics, non-constructive framing/rethorics or anecdotal arguments.
```

Comment:
rethorics -> rhetorics

I think in the case of issue reporting describing a personal experience (anecdote) is reasonable. So perhaps this could just be `...such as politics or non-constructive rhetoric.`

Reply:
You're right. I mean things like issue 873 with very long personal texts (don't want to link to it) or this recent comment "I don't have Albert launcher installed right now, but based on previous experience with using Albert I would expect all three of these apps to show for either of the search keywords mentioned above."

But people have to be able to speak from their perspective. I'll have to think about this.

Reply:
But I just removed it so we can get the ball rolling